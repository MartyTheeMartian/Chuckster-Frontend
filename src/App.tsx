import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
import Main from 'components/Main';
import Sidebar from 'components/Sidebar';
import Footer from 'components/Footer';
import './App.css';


export default function App(): JSX.Element {
  return (
    <div className='app-ctn'>
      <Router>
        <Sidebar />
        <Routes>
          <Route
            path='/'
            element={<Main />}
          />
        </Routes>
        <Footer />
      </Router>
    </div>
  );
};
