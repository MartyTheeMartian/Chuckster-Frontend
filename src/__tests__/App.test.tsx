import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from 'App';

it('renders App component', () => {
  const result = shallow(<App />);
  expect(toJson(result)).toMatchSnapshot();
});
