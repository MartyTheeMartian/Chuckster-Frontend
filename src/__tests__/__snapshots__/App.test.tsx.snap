// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App component 1`] = `
<div
  className="app-ctn"
>
  <BrowserRouter>
    <SideBar />
    <Routes>
      <Route
        element={<Main />}
        path="/"
      />
    </Routes>
    <Footer />
  </BrowserRouter>
</div>
`;
